test/integration/rpc.spec.ts

```diff
@@ -86,6 +86,7 @@ describe("rpc", () => {
 
     const [, result] = await Promise.all([server(), client()]);
     t.equal(
+      // @ts-ignore
```

Comment:
This turned up after the TS upgrade. I suspect this can be properly fixed, but I'm not super familiar with how the RPC system is meant to work, and so given this was a test I just put in a `@ts-ignore`